### PR TITLE
New version of BillingRP SDKs for API version 2019-10-01-preview

### PR DESCRIPTION
**Pull request by asarkar84**
New version of BillingRP SDKs for API version 2019-10-01-preview

### CODE REVIEW

**Review by isra-fel — CHANGES_REQUESTED** [review]
Hi @asarkar84 the generated sdk should be covered properly by test cases

**Comment by asarkar84** [comment]
Hi @isra-fel, We have written the test cases for most of the scenarios. Can you point out what is not done properly so that we can take action.

**Review by isra-fel — CHANGES_REQUESTED** [review]
Please also update the <PackageReleaseNotes> field in sdk/billing/Microsoft.Azure.Management.Billing/src/Microsoft.Azure.Management.Billing.csproj
Thanks

File: sdk/billing/Microsoft.Azure.Management.Billing/src/Microsoft.Azure.Management.Billing.csproj [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 	<PropertyGroup>
 		<Description>Microsoft Azure Billing Management Library</Description>
-		<Version>2.1.1-preview</Version>
+		<Version>3.1.1-preview</Version>
```
[the remark this answers is not in the document]
Comment (asarkar84) [reply sdk/billing/Microsoft.Azure.Management.Billing/src/Microsoft.Azure.Management.Billing.csproj]: Sure! I have incorporated the review comment.

File: sdk/billing/Microsoft.Azure.Management.Billing/src/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -8,8 +8,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Billing Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Billing.")]
 
-[assembly: AssemblyVersion("2.0.0.0")]
-[assembly: AssemblyFileVersion("2.1.1.0")]
+[assembly: AssemblyVersion("3.0.0.0")]
+[assembly: AssemblyFileVersion("3.1.1.0")]
```
[the remark this answers is not in the document]
Comment (asarkar84) [reply sdk/billing/Microsoft.Azure.Management.Billing/src/Properties/AssemblyInfo.cs]: Sure! I have incorporated the review comment.

**Comment by asarkar84** [comment]
@isra-fel  Thank you! for the review comments. We have incorporated the changes and sent the new iteration.

**Review by isra-fel — APPROVED** [review]

**Pull request merged**